hack/after-build/update-generated-conversions.sh

```diff
@@ -39,13 +39,23 @@ EOF
 
   "${genconversion}" -v "${version}" -f - >>  "$TMPFILE"
 
-	mv "$TMPFILE" "pkg/${version}/conversion_generated.go"
+    #TODO: remove this case if we rename the directory expapi/ to experimental/
```

Comment:
When you do that, I will recommend moving it to pkg/api/experimental. All groups in future like users, admin, etc can come under pkg/api (pkg/api/admin, pkg/api/users, pkg/api/experimental).